<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1263</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1263</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionCreate_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionAddFile"/>
    <addaction name="separator"/>
    <addaction name="actionSetting"/>
   </widget>
   <widget class="QMenu" name="menuModel_M">
    <property name="title">
     <string>Model(&amp;M)</string>
    </property>
    <addaction name="actionCreate_Model"/>
    <addaction name="actionSave_Model"/>
    <addaction name="actionManagement"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuModel_M"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionAddFile">
   <property name="text">
    <string>Add File</string>
   </property>
  </action>
  <action name="actionCreate_Project">
   <property name="text">
    <string>Create Project</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionCreate_Model">
   <property name="text">
    <string>Create Model</string>
   </property>
  </action>
  <action name="actionManagement">
   <property name="text">
    <string>Management</string>
   </property>
  </action>
  <action name="actionSave_Model">
   <property name="text">
    <string>Save Model</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
